src/app/components/JumpTo/index.stories.tsx

```diff
@@ -4,10 +4,24 @@ import JumpTo, { JumpToProps } from '.';
 import metadata from './metadata.json';
 import readme from './README.md';
 
-const Component = ({ jumpToHeadings = [] }: JumpToProps) => {
-  return <JumpTo jumpToHeadings={jumpToHeadings} />;
+const Component = ({
+  jumpToHeadings = [],
+  showRelatedContentLink = false,
+}: JumpToProps) => {
+  return (
+    <JumpTo
+      jumpToHeadings={jumpToHeadings}
+      showRelatedContentLink={showRelatedContentLink}
+    />
+  );
 };
 
+const jumpToBlock =
+  pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
+    block => block.type === 'jumpTo',
+  );
+
+const jumpToHeadings = jumpToBlock?.model.jumpToHeadings ?? [];
```

Comment:
Is it toooooo tricky / unreadable to destructure and do this all as a one-liner?

```suggestion
const { model: { jumpToHeadings } } =
  pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
    block => block.type === 'jumpTo',
  );
```

Reply:
Typescript also does not like this 🙅